javascript/javascript3/week3/ShoppingCart/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Some reviewers say : 
Try also to practice putting properties in alphabetical order, like
* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}